README.md

```diff
@@ -220,6 +220,144 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
     msvcp140d_oss.dll
 ```
 
+# How To Run The Tests From The Developer Command Prompt For VS
+
+1. Follow steps 1-9 of
+[How To Build With A Native Tools Command Prompt](#how-to-build-with-a-native-tools-command-prompt).

```

Comment:
Can these be the cool `[meow][]` form, or does that not work with internal section links?

Reply:
Maybe it does? I'll try it.

Reply:
Looks like it does.